pkg/auth/handler/webapp/authflow_controller.go

```diff
@@ -187,7 +187,7 @@ func (c *AuthflowController) HandleOAuthCallback(w http.ResponseWriter, r *http.
 		return
 	}
 
-	s, err := c.getWebSession(r)
+	s, err := c.Sessions.Get(state.WebSessionID)
```

Comment:
This is a bug which I think also affects authflow v1. If we are not going to deploy v2 onto staging shortly, maybe we need to pick this fix to main.

Reply:
I could not reproduce this bug on latest of authflowv2

Reply:
I tested again but cannot find how to reproduce the bug. Maybe I will drop this commit first.